source/deployment/ldap-group-constrained-team-channel.rst

```diff
@@ -0,0 +1,61 @@
+.. _ldap-group-constrained-team-channel:
+
+Using AD/LDAP Synchronized Groups to Manage Team or Private Channel Membership
+===============================================================
+Mattermost groups created with synchronized AD/LDAP groups can be used to manage the membership of private teams and private channels. When a team or private channel is managed by synchronized groups, users will be added and removed based on their membership to the synchronized AD/LDAP group.  
```

Comment:
Propose linking to AD/LDAP groups doc (in case someone lands directly on this page)

Reply:
Done.